packages/integrations/src/extraerrordata.ts

```diff
@@ -53,17 +56,22 @@ export class ExtraErrorData implements Integration {
    * TODO (v8): Drop this public function.
    */
   public enhanceEventWithErrorData(event: Event, hint: EventHint = {}): Event {
-    return _enhanceEventWithErrorData(event, hint, this._options.depth);
+    return _enhanceEventWithErrorData(event, hint, this._options.depth, this._options.captureErrorCause);
   }
 }
 
-function _enhanceEventWithErrorData(event: Event, hint: EventHint = {}, depth: number): Event {
+function _enhanceEventWithErrorData(
+  event: Event,
+  hint: EventHint = {},
```

Comment:
```suggestion
  hint: EventHint | undefined = {},
```

Reply:
Optional values should be at the end of the arguments list. I thought we had a rule for this?

Reply:
The parameter is not optional. It just takes undefined and falls back to `{}`. Optional parameters are marked with a `?` like this `hint?: EventHint`. Additionally, your suggestion doesn't do anything because TS already knows to allow undefined as type for parameters with default.